forest/sycamore/tests/.ipynb_checkpoints/Untitled-checkpoint.ipynb

```diff
@@ -0,0 +1,6 @@
+{
+ "cells": [],
+ "metadata": {},
+ "nbformat": 4,
+ "nbformat_minor": 5
+}
```

Comment:
Does this file need to be under version control?

Reply:
Nope, I didn't realize I had added it. Sorry about that